### PR TITLE
add sentry

### PR DESCRIPTION
**Pull request by km1230**

### CODE REVIEW

File: example.env [hunk 1]
```diff
@@ -34,3 +34,6 @@ MAILGUN_API_KEY="changeme"
 
 # Optional settings
 MAILGUN_SENDER_DOMAIN="mailgun.my_site.com"
+
+# Sentry
+SENTRY_DSN="changeme"
```
Comment (nattyg93) [inline example.env]: Can you please ensure the trailing newline is present

File: pyproject.toml [hunk 2]
```diff
@@ -29,6 +29,7 @@ gunicorn = "^19.9"
 psycopg2 = "^2.8"
 python-dateutil = "^2.8"
 pytz = "^2021.1"
+sentry-sdk = "^0.20.2"
```
Comment (nattyg93) [inline pyproject.toml:32]: Please ensure the dependencies are sorted.

**Review by nattyg93 — CHANGES_REQUESTED** [review]
Only need to make 2 changes. :)

File: pyproject.toml (hunk 2 above)
Comment (km1230) [reply pyproject.toml:32]: Ummm. It is...

**Pull request merged**